tests/features/asset_distribution/api.feature

```diff
@@ -0,0 +1,58 @@
+@api
+Feature: Solution API
+  In order to manage solutions programmatically
+  As a backend developer
+  I need to be able to use the Solution API
+
+  Scenario: Programmatically create a solution
+    Given the following collection:
+      | name              | Asset distribution collection API foo                  |
+      | logo              | logo.png                                               |
+      | moderation        | 1                                                      |
+      | closed            | 1                                                      |
+      | elibrary creation | facilitators                                           |
+      | uri               | http://joinup.eu/collection/asset_distribution-api-foo |
```

Comment:
Well, all in all, this is just to check the API. If the behat is all about user experience, maybe we should remove the API tests as they are not UATable.